File: src/main/cpp/algorithm/basic/BOJ_2309.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main(void){

    int allP[9];
    int result = 0;
    for(int i = 0; i < 9; i++){
        cin >> allP[i];
        result += allP[i];
    }
    sort(allP, allP+9);

    for(int i = 0; i < 8; i++){
        for(int j = 0; j < 9; j++){
            if(result - (allP[i] + allP[j]) == 100){
                for(int k = 0; k < 9; k++){
                    if(k != i && k != j ){
                        cout << allP[k] << "\n";
                    }
                }
                return 0;
            }
        }
    }

    return 0;
}

File: src/main/java/com/algorithm/bruteforce/BOJ_2309.java
package com.algorithm.bruteforce;

import java.io.*;
import java.util.*;

public class BOJ_2309 {

    public static void main(String[] args) throws IOException{

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int[] arr = new int[9];
        int result = 0;
        for(int i = 0; i < 9; i++){
            arr[i] = Integer.parseInt(br.readLine());
            result += arr[i];
        }

        Arrays.sort(arr);

        for(int i = 0; i < 8; i++){
            for(int j = 0; j < 9; j++){
                if(result-(arr[i]+arr[j]) == 100){
                    for(int k = 0; k < 9; k++){
                        if(k != i && k != j){
                            System.out.println(arr[k]);
                        }
                    }
                    return;
                }
            }
        }

    }

}
